### ISSUE TITLE
obs64 process doesn't close when quitting app on macos

### ISSUE THREAD

**Issue by alse** [Envek/obs-studio-node-example#22]
Tested on #19. It also happens with the streamlabs app in dev mode, but not the production release.

Is anyone seeing this on windows as well?

**Comment by hrueger** [comment]
Do you have any extra preview windows / displays? I needed to explicitly close them before shutting down obs (but I'm running windows).

**Comment by alse** [comment]
No, they're all closed. I tried skipping the scene & preview setup completely, so that the app only initializes and shuts down. No luck

**Comment by chenyangxx** [comment]
same question
